tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES6",
+    "target": "ESnext",
```

Comment:
This might introduce feature not support by node. It's fine as long as we always test new tsc versions with our production node version (which we do in CI/CD) but we do need to be aware of this. If it's just for async/await then ES2017 should surfice.